server/@types/manageAndDeliverApi/index.d.ts

```diff
@@ -50,6 +50,7 @@ type Group = components['schemas']['Group']
 type ProgrammeGroupCohortEnum = components['schemas']['CreateGroup']['cohort']
 type ProgrammeGroupDate = components['schemas']['CreateGroup']['startedAtDate']
 type ProgrammeGroupSexEnum = components['schemas']['CreateGroup']['sex']
+type ProgrammeGroupWhen = components['schemas']['CreateGroup']['createGroupSessionSlot']
```

Comment:
Nothing critical, but may be good to keep the naming consistent between the ui and api if using the same type.
That being 'createGroupSessionSlot